ee/agent/keys.go

```diff
@@ -40,11 +39,6 @@ func SetupKeys(slogger *slog.Logger, store types.GetterSetterDeleter) error {
 		return fmt.Errorf("setting up local db keys: %w", err)
 	}
 
-	// Secure Enclave is not currently supported, so don't spend startup time waiting for it to work -- see keys_darwin.go for more details.
```

Comment:
should we roll this out internally only first?

Reply:
Are there risks associated with rolling it out?

Reply:
I remember we had the issue with some beta versions causing trouble, but that might have been because it was failing over and over again. I think the suggestion about adding traces to this area was a good one and will give more confidence.

Reply:
IIRC one version of macOS threw ugly errors with the secure enclave call, and caused launcher to crash